src/Core/TileGeometry.js

```diff
@@ -222,12 +225,10 @@ TileGeometry.prototype.computeBuffers = function computeBuffers(params, builder)
     var iStart = idVertex;
 
     // TODO: WARNING beware skirt's size influences performance
-    // Fix Me: Compute correct the skirt's size : minimize the size without crack between tiles
-    // This size must be take into account the bbox's size
-    // For the moment, I reduce the size to increase performance (pixel shader performance)
-
+    // The size of the skirt is now a ration of the size of the tile.
+    // To be perfect it should depend on the real elevation delta but too heavy to compute
     if (!params.disableSkirt) {
-        const r = 5 * ((20 - params.level) + 10);
+        const r = segmentSize; // Tile size divided by the number of segments
```

Comment:
Using segment size allows the ration skirt size/tile size to stay the same at each level of the tree.
The previous simplistic formula doesn't do that and produce huge skirts (heavy frag shading) when close to the ground and very small when high in sky (creating holes)